styles/style.css

```diff
@@ -0,0 +1,123 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+
+* {
+    font-family: 'Roboto', sans-serif;
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+h1 {
+    display: flex;
+    justify-content: center;
+    width: 306px;
+    height: 47px;
+    margin: 120px 567px 0 567px;
```

Comment:
Unneccessary, you can yse paading top and thats all

Reply:
done